.env

```diff
@@ -34,7 +34,34 @@ ILIOS_DATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVers
 # Set a default upload limit to match AWS's non-huge ES service limits
 ILIOS_ELASTICSEARCH_UPLOAD_LIMIT=8000000
 
-# For Gmail as a transport, use: "gmail://username:password@localhost"
-# For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
-# Delivery is disabled by default via "null://localhost"
-ILIOS_MAILER_URL=null://localhost
+###> symfony/mailer ###
+MAILER_DNS=null://null
+###< symfony/mailer ###
+
+###> symfony/amazon-mailer ###
+# MAILER_DSN=ses://ACCESS_KEY:SECRET_KEY@default?region=eu-west-1
+# MAILER_DSN=ses+smtp://ACCESS_KEY:SECRET_KEY@default?region=eu-west-1
+###< symfony/amazon-mailer ###
+
+###> symfony/google-mailer ###
+# Gmail SHOULD NOT be used on production, use it in development only.
```

Comment:
Given this, I think we shouldn't add support for the google-mailer.

Reply:
google mailer has been yanked.